compiler/rustc_const_eval/src/interpret/discriminant.rs

```diff
@@ -2,12 +2,28 @@
 
 use rustc_middle::mir;
 use rustc_middle::ty::layout::{LayoutOf, PrimitiveExt};
-use rustc_middle::ty::{self, Ty};
+use rustc_middle::ty::{self, ScalarInt, Ty};
 use rustc_target::abi::{self, TagEncoding};
 use rustc_target::abi::{VariantIdx, Variants};
 
 use super::{ImmTy, InterpCx, InterpResult, Machine, Readable, Scalar, Writeable};
 
+/// The tag of an enum discriminant.
+pub(crate) enum Tag {
```

Comment:
This enum is only useful for writing the tag, not for reading it. I feel like that should somehow be reflected? At least in the docs, maybe in the name. In MiniRust we'd call this the "tagger" for this variant.

Reply:
Done, by removing `Tag` altogether.